linera-core/src/client/mod.rs

```diff
@@ -1807,15 +1816,26 @@ where
         for blob_id in blob_ids {
             let mut certificate_stream = validators
                 .iter()
-                .map(|remote_node| remote_node.download_certificate_for_blob(blob_id))
+                .map(|remote_node| async move {
+                    let cert = remote_node.download_certificate_for_blob(blob_id).await?;
```

Comment:
Does this still keep the old semantics of running all futures in parallel thanks to `async move`?

Reply:
Yes, it maps each `remote_node` to a future (the `async move` block) and then creates the `FuturesUnordered` from them.